cmd/celestia/blob.go

```diff
@@ -0,0 +1,189 @@
+package main
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"github.com/celestiaorg/celestia-node/blob"
+	"github.com/celestiaorg/celestia-node/share"
+)
+
+type response struct {
+	Result interface{} `json:"result"`
+}
```

Comment:
very optional nit: move it to where its used - close to `prepareOutput`

Reply:
can even insert it inside prepareOutput